doc/nrf/create_application.rst

```diff
@@ -0,0 +1,212 @@
+
+.. _create_application_types_freestanding:
+
+Freestanding application
+========================
+
+A freestanding application is an application that is not inside a west workspace and that is not using the `west manifest file`_ to specify the |NCS| version.
+Instead, the |NCS| version specified in the workspace setting is used.
+This means that all freestanding applications in the same workspace will use the same |NCS| version.
+
+.. code-block:: none
+
+   <home>/
+   ├─── ncs/
+   │  └── ...
+   └─── <app>/
+      ├── CMakeLists.txt
+      ├── app.overlay
+      ├── prj.conf
+      └── src/
+        └── main.c
+
+You want to use this application type for the following development cases:
+
+* You are not planning to make any changes to any of the repositories of the |NCS| when working on the application.
+
+For more information about freestanding applications in the |NCS|, see the :ref:`development model page <dm_workflow_2>`.
+
+Creating an |NCS| application
+*****************************
+
+The process for creating an |NCS| application depends the development environment.
+
+.. _creating_vsc:
+
+Creating application in the |nRFVSC|
+====================================
+
+Use the following steps depending on the application type:
+
+.. tabs::
+
+   .. group-tab:: Workspace application (recommended)
+
+      To create a workspace application in the |nRFVSC|:
+
+      1. Open |VSC|.
+      #. Open the |nRFVSC|.
+      #. In the :guilabel:`Welcome View`, click the :guilabel:`Create a new application` action.
+         The New Application page opens.
+      #. Select :guilabel:`Workspace` as the application type.
+      #. Configure other options depending on what application you want to create.
+         See tooltips for more information.
+         If you want to create the most basic application, select ``zephyr/hello_world`` in the :guilabel:`Application template` field.
+      #. Click :guilabel:`Create Application`.
+
+      See the `extension documentation <Create a new application_>`_ for more information.
+
+   .. group-tab:: Freestanding application
+
+      To create a freestanding application in the |nRFVSC|:
+
+      1. Open |VSC|.
+      #. Open the |nRFVSC|.
+      #. In the :guilabel:`Welcome View`, click the :guilabel:`Create a new application` action.
+         The New Application page opens.
+      #. Select :guilabel:`Freestanding` as the application type.
+      #. Configure other options depending on what application you want to create.
+         See tooltips for more information.
+         If you want to create the most basic application, select ``zephyr/hello_world`` in the :guilabel:`Application template` field.
+      #. Click :guilabel:`Create Application`.
+
+      See the `extension documentation <Create a new application_>`_ for more information.
+
+You can now start :ref:`configuring and building <configuration_and_build>` the application.
+
+.. _creating_cmd:
+
+Creating application for use with command line
```

Comment:
Please link here:
https://docs.zephyrproject.org/latest/develop/application/index.html#creating-an-application-by-hand

Reply:
The page in Zephyr does not have header tags for that section, so I've added a link to the entire page and tried to clarify where to look in the link title.

Reply:
> The page in Zephyr does not have header tags for that section, so I've added a link to the entire page and tried to clarify where to look in the link title.

We should at header tags upstream instead.

Reply:
I will do that, but until the next upmerge, we'll have to link to the entire page.

Reply:
@wbober , @carlescufi , @tmon-nordic , could you check and review/approve? https://github.com/zephyrproject-rtos/zephyr/pull/61058